src/mach/header.rs

```diff
@@ -232,21 +217,6 @@ unsafe impl Plain for Header64 {}
 
 pub const SIZEOF_HEADER_64: usize = 32;
 
-impl fmt::Debug for Header64 {
```

Comment:
Why remove the debug ? Instead of making it proper with debug struct, etc?

I find it very useful especially as when I’m testing emitting mach objects

Reply:
@m4b heh. I guess that's personal preference. I switched to the derived `Debug` when debugging because I was comparing to apple tools and I got confused by the output a bit.

Reply:
(I did not change the higher level Header debug output other than that I removed the cpu name printing)